### ISSUE TITLE
1.15.2 ticking arrow entity 

### ISSUE THREAD

**Issue by ArMoReDfUrY** [DeflatedPickle/Heart-Drops#14]
hello :) been a while.

got another one for you, had it for a while now but have been busy with a few world breakers in other mods.
heres the paste:  https://pastebin.com/ZqKvsebA

this is the line the crash refs:
https://github.com/DeflatedPickle/HeartDrops/blob/92d73c6b7facecb4b43603439933407db4ee9ff2/src/main/kotlin/com/deflatedpickle/heartdrops/event/ForgeEventHandler.kt#L77

it appears entities firing arrows can be problematic...

**Comment by DeflatedPickle** [comment]
That is true, it's possible the event is running but it's from a skeleton shooting? This could be why it's coming up as `null`. I'll add a `null` check!
